app/scripts/controllers/network/network-controller.test.ts

```diff
@@ -1795,51 +1595,47 @@ describe('NetworkController', () => {
                   },
                 },
               },
-              async ({ controller, network }) => {
-                network.mockEssentialRpcCalls({
-                  // Ensure that each call to eth_blockNumber returns a
```

Comment:
This is a nice side effect of using a fake provider as well — we don't need to worry about mocking `eth_blockNumber` in these tests because the block tracker never gets started.